boards/arc/qemu/doc/index.rst

```diff
@@ -0,0 +1,100 @@
+.. _qemu_arc:
+
+ARC EM & HS Emulation (QEMU)
+###############################
+
+Overview
+********
+
+This board configuration will use QEMU to emulate a generic ARC EM &
+ARC HS hardware platform.
+
+The following features of ARCv2 ISA cores are currently supported:
+
+* ARC EM or ARC HS core (both belong to ARCv2 ISA family still
+  they are binary incompatible)
+* Only little-endian configurations
+* Full 32 register set
+* ARC core free-running timers/counters Timer0 & Timer1
+* ARC core interrupt controller with multiple priority levels
+* DW UART
+* 5 slots for MMIO Virtio devices
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| ARCv2 INTC   | on-chip    | interrupt controller |
++--------------+------------+----------------------+
+| DW UART      | on-chip    | serial port          |
++--------------+------------+----------------------+
+| ARC TIMER0   | on-chip    | system clock         |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 1 MHz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+DesignWare UART.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Memory-protection unit (MPU)
+
+Programming and Debugging
+*************************
+
+Use this configuration to run basic Zephyr applications and kernel tests in the QEMU
+emulated environment, for example, with the :ref:`synchronization_sample`
+(note you may use ``qemu_arcem`` or ``qemu_archs`` depending on target CPU):
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/synchronization
+   :host-os: unix
+   :board: qemu_arcem
+   :goals: run
+
+This will build an image with the synchronization sample app, boot it using
+QEMU, and display the following console output:
+
+.. code-block:: console
+
+        *** Booting Zephyr OS build zephyr-v2.2.0-2486-g7dbfcf4bab57  ***
+        threadA: Hello World from qemu!
+        threudB: Hello World from qemu!
+        threadA: Hello World from qemu!
+        threadB: Hello World from qemu!
+
+Exit QEMU by pressing :kbd:`CTRL+A` :kbd:`x`.
+
+Debugging
+=========
+
+Refer to the detailed overview about :ref:`application_debugging`.
+
+Networking
```

Comment:
This heading should be removed I guess..

Reply:
Yet another copy-paste from ARM Cortex A53 docs. So frankly I have mo idea what's the right contents here, thus we're reviewing it here :)

Any good example or guide to look at?

Reply:
All content looks okay.. I'm only pointing to remove the "Networking" heading because its not supported in your case. For reference, you can take a look at the `qemu_x86` readme file..

Reply:
Well as a matter of fact we may add networking with help of Virtio Ethernet controller - it's proven to work perfectly fine with Linux. But it looks like Virtio Ethernet controller driver is not yet implemented in the Zephyr, right? So then I'm dropping that header. Thanks for spotting this one!